### ISSUE TITLE
tBTC dApp: Freezing of the page after completing step 3

### ISSUE THREAD

**Issue by nikgrego** [keep-network/tbtc-dapp#265]
### Description:
Since the confirmation of the Bitcoin transaction takes a long time, on the step 3, while dApp is waiting for confirmation of BTC deposit - the page does not recognize transaction confirmation and does not react after, you need to manually refresh the page and connect to a wallet again, only after doing this you can move to step 4.

### Steps to reproduce the behavior:
1. Go to https://dapp.test.tbtc.network
2. Pass all the steps from 1 to 3

### Expected behavior:
Completion of the last step without page refresh, or at least without re-connecting your wallet again.

### Environment details:
OS: OS X El Capitan, 10.11.6 (15G22010)
Web Browser: Google Chrome, Version 83.0.4103.116 (Official Build) (64-bit)  
Wallet: Metamask (Chrome extension), Version 8.0.3

### Screenshots
![Скриншот 2020-07-13 13 48 14](https://user-images.githubusercontent.com/66184067/87547979-8cfc9d80-c6b4-11ea-8e0a-01a0662f2d9f.png)

**Comment by Shadowfiend** [comment]
> Since the confirmation of the Bitcoin transaction takes a long time, on the step 3, while dApp is waiting for confirmation of BTC deposit - the page does not recognize transaction confirmation

Interesting! This should not be the case… Did you change connectivity during the confirmation (e.g., switch networks, close laptop/reopen laptop, etc)? We may be doing a poor job of reconnecting in those cases, which leads to the underlying issue.

**Comment by nikgrego** [comment]
> > Since the confirmation of the Bitcoin transaction takes a long time, on the step 3, while dApp is waiting for confirmation of BTC deposit - the page does not recognize transaction confirmation
> 
> Interesting! This should not be the case… Did you change connectivity during the confirmation (e.g., switch networks, close laptop/reopen laptop, etc)? We may be doing a poor job of reconnecting in those cases, which leads to the underlying issue.

nope, the only thing i did - was switching to other tab in Chrome with block.io to check the status of transaction
